recipes/seqan-library/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "seqan-library" %}
+{% set version = "2.2.0" %}
+{% set sha256 = "b5c036a3d2fc2fe5f2d57dcd4d7c523a6df146ab7b44bc789f42004b058d72fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://github.com/seqan/seqan/releases/download/seqan-v{{ version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: cp -r include share $PREFIX  # [unix]
+
+requirements:
+  build:
+    - python >=3.3  # [win]
```

Comment:
ah, that is what I want. In the last release seqan published only a tar.xz release and conda with py27 does not support that on windows. Apparently they'll fix that and provide a zip, too, but for the time being I thought I'll just skip for py<33.